.github/renovate.json

```diff
@@ -47,23 +116,132 @@
       "enabled": false,
-      "matchDatasources": ["npm"],
-      "matchPackageNames": ["@keycloak/keycloak-admin-client"],
+      "matchDatasources": [
+        "npm"
+      ],
+      "matchPackageNames": [
+        "@keycloak/keycloak-admin-client"
+      ],
       "groupName": "Keycloak dependency"
     },
     {
       "description": "ignore updates to the axios to version that keycloak 18 needs",
       "enabled": false,
-      "matchDatasources": ["npm"],
-      "matchPackageNames": ["axios"],
+      "matchDatasources": [
+        "npm"
+      ],
+      "matchPackageNames": [
+        "axios"
+      ],
       "matchCurrentVersion": "0.26.1",
       "groupName": "Keycloak dependency"
+    },
+    {
+      "matchFileNames": [
+        "plugins/orchestrator*/**"
+      ],
+      "additionalBranchPrefix": "orchestrator ",
+      "addLabels": [
+        "team/orchestrator"
+      ]
+    },
+    {
+      "matchFileNames": [
+        "plugins/notifications*/**"
+      ],
+      "additionalBranchPrefix": "notifications ",
+      "addLabels": [
+        "team/notifications"
+      ]
+    },
+    {
+      "matchFileNames": [
+        "plugins/kiali*/**"
+      ],
+      "additionalBranchPrefix": "kiali ",
+      "addLabels": [
+        "team/kiali"
+      ]
+    },
+    {
+      "matchFileNames": [
+        "plugins/*matomo*/**",
+        "plugins/analytics-module-matomo*/**",
+        "plugins/feedback*/**"
+      ],
+      "additionalBranchPrefix": "devex ",
+      "addLabels": [
+        "team/devex"
+      ]
+    },
+    {
+      "matchFileNames": [
+        "plugins/quay*/**",
+        "plugins/tekton*/**",
+        "plugins/argocd*/**"        
+      ],
+      "additionalBranchPrefix": "rhtap ",
+      "addLabels": [
+        "team/rhtap"
+      ]
+    },
+    {
+      "matchFileNames": [
+        "plugins/analytics-provider-segment*/**",        
+        "plugins/bulk-import*/**",    
+        "plugins/dynamic-plugins-info*/**",
+        "plugins/keycloak*/**",
+        "plugins/ocm*/**",
+        "plugins/quay-actions*/**",
+        "plugins/rbac*/**",
+        "plugins/regex-actions*/**",
+        "plugins/scaffolder-annotator-action*/**",          
+        "plugins/topology*/**",
+        "packages/**"
+      ],
+      "additionalBranchPrefix": "rhdh ",
+      "addLabels": [
+        "team/rhdh"
+      ]
+    },
+    {
+      "matchFileNames": [ 
+        "package.json"  
+      ],
+      "additionalBranchPrefix": "rhdh ",
+      "addLabels": [
+        "team/rhdh"
+      ]  
+    },
+    {
+      "matchFileNames": [
+        "plugins/3scale-backend*/**",
+        "plugins/aap-backend*/**",
+        "plugins/acr*/**",
+        "plugins/jfrog-artifactory*/**",
+        "plugins/kubernetes-actions*/**",
+        "plugins/nexus-repository-manager*/**",
+        "plugins/openshift-image-registry*/**",
+        "plugins/servicenow-actions*/**",
+        "plugins/shared-react*/**",
```

Comment:
I think this is a supported one, as tekton and topology depend on it. https://github.com/janus-idp/backstage-plugins/pull/1627/files#diff-893725eef4377bb0ab545442ef744d25e17a4ebb023fbf945598984cb6a03bfdR25

see also https://docs.google.com/spreadsheets/u/1/d/1dNUAGTeosEzreX9fxFTW1Vq3oGCbHroZ_DFEL8sjfjY/edit#gid=0 line 112 "depended on by topology, tekton, quay, openshift-image-registry, nexus-repository-manager, jfrog-artifactory, acr"